language/move-prover/tests/sources/functional/loops.move

```diff
@@ -0,0 +1,86 @@
+module VerifyLoops {
+
+    spec module {
+        pragma verify=false;
+    }
+
+    // ----------------------
+    // `aborts_if` statements
+    // ----------------------
+
+    public fun iter10_missing_inc_spec1() {
+        let i = 0;
+        while (i <= 10) { // an infinite loop
+            if (i > 10) abort 10;
+        }
+    }
+    spec fun iter10_missing_inc_spec1 { // TODO: What is the expected output for this spec?
+        aborts_if false;
+    }
+
+    public fun iter10_missing_inc_spec2() {
+        let i = 0;
+        while (i <= 10) { // an infinite loop
+            if (i > 10) abort 10;
+        }
+    }
+    spec fun iter10_missing_inc_spec2 { // TODO: What is the expected output for this spec?
```

Comment:
This should also verify.  It really does not matter what postcondition you write. They will all verify.

Reply:
Got it. Our spec asserts the partial correctness of the functions, so any spec is supposed to be simply true when the function contains an infinite loop does not terminate?